src/pages/welcome/Welcome.tsx

```
@@ -1,35 +1,43 @@
       <Card className="fullpage-content">
-        <div style={{ marginTop: '20px', display: 'flex', justifyContent: 'center' }}>
+        <div className={styles.fullpage}>
           <div>
-            <H2>Welcome to {Constants.sourceAcademyDeploymentName}</H2>
+            <H2>{t('welcome.title', { sourceAcademyDeploymentName })}</H2>
             <div>
-              You have logged in as <strong>{name}</strong>. {Constants.sourceAcademyDeploymentName}{' '}
-              does not have any course information for this account.
+              <Trans
+                i18nKey="welcome.loggedInMessage"
+                components={[<strong />]}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)